### ISSUE TITLE
During checkout: when SR users adds new Payment Method more context is needed.

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13733]
**Describe the bug**
- When adding a new Payment Method during checkout, the checkbox "Set as default" needs more context. E.g. "Set this payment method as default"
- If user tabs to checkbox "Same as shipping address" only **hears** exactly that. Without the missing context of "billing address is the same as shipping address

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**

1. Steps to reproduce the behavior:
2. Log in as a registered user
3. Add items to cart and start checkout process.
4. Open any Screen Reader (VO, NVDA, JAWS).
5. Go to the Payment Details step.
6. Hit the Add New Payment button
7. Tab through the form, focus on checkboxes
8. Listen to screen reader feedback

**Expected behavior**
- Use `aria-label` to extend screen reader vocalization, as per screenshots below.
- - The `aria-label` is not needed if designers agree to change the text for mouse users as well, please validate with @Manurod2018 and @vekterdude 

**Screenshots**
Suggested vocalization:

![Screen Shot 2021-09-15 at 11 33 05 AM](https://user-images.githubusercontent.com/15113219/133469140-7421e749-b98f-478a-b976-edac0af9d7ab.png)
![Screen Shot 2021-09-15 at 11 33 44 AM](https://user-images.githubusercontent.com/15113219/133469190-39fef54a-b995-4042-b5c6-806e4e77b213.png)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus. 


![payment_1](https://user-images.githubusercontent.com/45409515/144293190-38b1528a-2af8-427b-9ae8-c0d2862f3bfe.png)
![payment_2](https://user-images.githubusercontent.com/45409515/144293195-6e6410c7-547e-41df-9fbb-01b175273650.png)
![payment_3](https://user-images.githubusercontent.com/45409515/144293200-d23467d8-3c35-403f-a184-6122a6c6821e.png)
![payment_4](https://user-images.githubusercontent.com/45409515/144293204-ddefd5f0-dd80-49a7-b474-32416a2c99f3.png)
![payment_5](https://user-images.githubusercontent.com/45409515/144293206-07c59977-7ce1-4685-8ae1-7512d36dfd0c.png)

More extensive testing (on other screenreaders and devices) will be done with RC.
